pagopa/io-app
# chore(IT Wallet): [SIW-962] Add cards onboarding CTA in the new wallet section

## Short description
This PR adds the card onboarding CTA in the new wallet section.

## List of changes proposed in this pull request
- Refactored `WalletHomeScreen` to allow the addition of the CTA
- Added `WalletHomeScreen` component tests

## How to test
From the **Profile** section, enable the **New wallet section** FF.
You should be able to see the CTA in the wallet screen

## Preview
<img src="https://github.com/pagopa/io-app/assets/6160324/cd039175-9526-4618-8b9f-3da3142c835d" width="200" />
